ghost/members-api/test/unit/lib/repositories/member.test.js

```diff
@@ -338,16 +338,26 @@ describe('MemberRepository', function () {
         });
 
         it('subscribes a member to a specified newsletter', async function () {
```

Comment:
```suggestion
        it('subscribes a member to the specified newsletters', async function () {
```

Now we have updated the functionality we should update the test name to reflect